integration/ejb/perftest/perf-eap5/pom.xml

```diff
@@ -23,7 +23,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.8.2</version>
+            <version>[4.13.1,)</version>
```

Comment:
Can we define the version in property, here and other modified pom.xml files?

Reply:
No because they don't all have parents (only one of them did).

Reply:
Okay

Reply:
If you want them all to inherit from a common parent where we would defined common properties you would need to raise a separate JBTM for that,